src/index.js

```diff
@@ -1,8 +1,12 @@
-var _       = require("lodash"),
+const _assign = require("lodash.assign"),
+    _toArray = require("lodash.toarray"),
+    _forEach = require("lodash.foreach"),
+    _concat = require("lodash.concat"),
+    _get = require("lodash.get"),
```

Comment:
I also don't love the underscores everywhere. The lodash docs seem to just recommend just calling it "get": https://www.npmjs.com/package/lodash.get.